### ISSUE TITLE
TypeError after upgrade to 2.0

### ISSUE THREAD

**Issue by jaraco** [jaraco/keyring#107]
Keyring 2.0 select `keyring.backends.Windows.RegistryKeyring` as default backend on Windows. I got the TypeError (traceback below) on Windows 7 64-bit with Python 3.3.2 32 bit. It looks like a bug in `_win_crypto.py` module.

`keyring.backends.file.PlaintextKeyring` and `keyring.backends.Windows.EncryptedKeyring` are working properly.

```
#!

Traceback (most recent call last):
  File "D:\Workspace\Marker\src\dialogs\settings.py", line 105, in accept
    mailSettingsSaved = self.mailPage.saveSettings()
  File "D:\Workspace\Marker\src\dialogs\settings.py", line 274, in saveSettings
    EmailSettings.setPasswd(passwd)
  File "D:\Workspace\Marker\src\settings.py", line 100, in setPasswd
    keyring.set_password("Email", cls.__email, value)
  File "D:\env\marker\lib\site-packages\keyring\core.py", line 42, in set_passwo
rd
    _keyring_backend.set_password(service_name, username, password)
  File "D:\env\marker\lib\site-packages\keyring\backends\Windows.py", line 188,
in set_password
    password_encrypted = _win_crypto.encrypt(password)
  File "D:\env\marker\lib\site-packages\keyring\backends\_win_crypto.py", line 6
8, in encrypt
    pbData=cast(c_char_p(data),
TypeError: bytes or integer address expected instead of str instance
```

---
- Bitbucket: https://bitbucket.org/kang/python-keyring-lib/issue/107
- Originally reported by: Krystian Rosiński
- Originally created at: 2013-08-20T18:35:06.368

**Comment by jaraco** [comment]
Very interesting. Thanks for the report. I see now that the RegistryKeyring isn't included in the test suite, and hence why it fails on a basic operation. I've got it in the test suite now with 12e0a8fd8f6b and I'll get the bugs fixed shortly.

---

Original comment by: Jason R. Coombs

**Comment by jaraco** [comment]
Fix failing tests (on Python 3). Tests are still failing on Python 2 because the cursed _winreg module doesn't support Unicode. Fixes #107.

→ <<cset 2eabb81746a2>>

---

Original comment by: Jason R. Coombs

**Comment by jaraco** [comment]
Thank you. 2.0.1 works for me.

---

Original comment by: Krystian Rosiński
